test-framework/vertx/src/main/java/io/quarkus/test/vertx/UniAsserter.java

```diff
@@ -6,53 +6,124 @@
 
 import io.smallrye.mutiny.Uni;
 
+/**
+ * Used in conjunction with {@link RunOnVertxContext} (or similar annotations) in order to test assertions
+ * on APIs that return {@link Uni} and whose execution needs to occur on a Vert.x Event Loop Thread (such as Hibernate Reactive
+ * for example).
+ *
+ * The following example shows how this class is meant to be used:
```

Comment:
Shouldn't we explicitly state that it must be used as a parameter of a test method?

Reply:
Good point

Reply:
Should be better now